themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -71,6 +55,7 @@ $screen-lg: "screen and (min-width: 1200px)";
 body {
     background-color: $bg-color;
     font: 400 1.5em $font-sans;
+	color: $font-p-color;
```

Comment:
I think you are mixing spaces and tabs.  The indentation between the CSS rules looks off.